nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -268,6 +268,15 @@ in
         };
       };
 
+      moduliFile = mkOption {
+        default = "${cfgc.package}/etc/ssh/moduli";
```

Comment:
Looks good, except that having store paths in defaults is problematic because it makes the manual depend on them. So it's better to have the default in the `config` section.